config/routes.rb

```diff
@@ -41,7 +41,7 @@
     end
 
     scope '/r/insights' do
-      match '(/*path)(/)', :constraints => lambda { |req| !req.path.include?('view/api') }, to: 'machine_telemetries#forward_request', via: [:get, :post, :delete,:put, :patch]
+      match '(/*path)(/)', :constraints => ->(req) { !req.path.include?('view/api') }, :to => 'machine_telemetries#forward_request', :via => [:get, :post, :delete, :put, :patch]
```

Comment:
Wouldn't it be better to make it consistent this way:
```suggestion
      match '(/*path)(/)', constraints: ->(req) { !req.path.include?('view/api') },  to: 'machine_telemetries#forward_request', via: [:get, :post, :delete, :put, :patch]
```

Reply:
consistent to what? the rest of the file uses both styles, so it's hard to say ;-)

(I don't mind either way, this is just what the :policeman: generated)

Reply:
I think my suggestion is a smaller diff and also more consistent within the file.